inventory/upcloud.ini

```diff
@@ -20,3 +20,4 @@
 return_ip_addresses = True
 return_non_fqdn_names = False
 default_timeout = 300
+default_ipv_version: = Ipv4
```

Comment:
Should this be IPv4? Like in `get_hostname_or_ip()`?

Reply:
Yes. This should be a string but this will be refactored with changes and wont exist since constants that are retrieved from ini will only be there.

Reply:
My bad, I got confused by the question. This will be the default ipv version to use. As in https://github.com/UpCloudLtd/upcloud-ansible/issues/3, the user will be able to change what preferred ipv version will be used